prdoc/pr_5007.prdoc

```diff
@@ -0,0 +1,11 @@
+title: Add experimental slot based collator to parachain template
+
+doc:
+  - audience: Node Dev
+    description: |
+      Adds an `--experimental-use-slot-based` CLI parameter to the default parachain template,
+      which enables the slot based collator.
```

Comment:
Must also mention that it should only be used to enable elastic scaling on the client.